ci-operator/step-registry/ipi/conf/aws/min-perms-user/ipi-conf-aws-min-perms-user-ref.yaml

```diff
@@ -2,7 +2,7 @@ ref:
   as: ipi-conf-aws-min-perms-user
   from_image:
     namespace: ocp
-    name: "4.14"
+    name: "4.18"
```

Comment:
Ah, I forgot aws sdk mentioned in https://github.com/openshift/release/pull/57893#discussion_r1804256715 .

If we would like the step work with different version of installer, from my understanding, the only way is to develop separate install workflow, then append this step, but not append it to `ipi-conf-aws` chain.

Reply:
Unless we separate it into 2 stages: 
1. Generate the policy. For this we only need `installer` and `jq`
2. Create the iam user and attach the policy. For this we only need the aws cli, so any upi-installer version suffices.